components/esp_lvgl_port/docs/performance.md

```diff
@@ -0,0 +1,125 @@
+# LCD & LVGL Performance
+
+This document provides steps, how to set up your LCD and LVGL port for the best performance and comparing the difference settings. All settings and measurements are valid for Espressif's chips.
+
+## Process of the performance measuring
+
+There are three main processes for measurement performance on LCD:
```

Comment:
Maybe: In this document we will use following metrics for performance evaluation: